content/pl/docs/preparation/macOS/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: MacOS
+linkTitle: MacOS
+weight: 1
+description: >
+    Opis przygotowania nośnika instalacyjnego z MacOS.
+---
+
+
+# Pobieranie instalatora z gibMacOS
+
+* ####  *Narzędzie to pozwoli Ci pobrać kopię MacOS w wersji od 10.13.x do najnowszej* 
+
+Jeżeli jesteś na maszynie spełniającej wymagania systemu który chcesz pobrać, możesz użyć App store do pobrania instalatora.
+
+Pobierz narzędzie [gibMacOS](https://github.com/corpnewt/gibMacOS), następnie rozpakuj, przejdź do folderu i odpal `gibMacOS.command`:
+
+![alt](gibmacos.png)
+
+Pojawią się wersje MacOS z katalogu publicznego, jeśli chcesz pobrać wersję Beta zmień katalog : `C.Change Catalog`
+
+* Kopia instalatora bedzie pobrana do katalogu `gibMacOS-master/macOS Downloads`
+* Aby przekonwertować pobrane pliki do `.app` i przenieść instalator do folderu /Applications uruchom `InstallESDDmg.pkg`
```

Comment:
To akurat chyba nie jest do końca prawda. Możemy tutaj użyć pliku https://github.com/corpnewt/gibMacOS/blob/master/BuildmacOSInstallApp.command, który stworzy za nas plik .app w katalogu z plikami .pkg

Reply:
obydwie metody dzialaja, robilem tak z pkg wiele razy

Reply:
Jednak zostałbym przy tej "prostszej" metodzie.